ETR01SDK-630: Docs: refactor CMake options mentions in tutorials (#537)
tropicsquare/libtropic: docs/tutorials/linux/spi/identify_chip.md

    After this, you should see an output in your terminal.

## Configuration
Besides the [Libtropic CMake options](../../../reference/integrating_libtropic/how_to_configure/index.md) used to configure Libtropic, there are multiple CMake options specific to this example:

> - `LT_SPI_DEV_PATH` (default: `"/dev/spidev0.0"`) to set the path to the SPI device where TROPIC01 is connected:

Reviewer: ```suggestion
- `LT_SPI_DEV_PATH` (default: `"/dev/spidev0.0"`) specifies the path to the SPI device to which the TROPIC01 is connected:
```

    ??? example "Configuring SPI device path"
        === ":fontawesome-brands-linux: Linux"
            ```bash { .copy }
            cmake -DLT_SPI_DEV_PATH=<spi_dev_path> ..
            make
            ./libtropic_identify_chip
            ```

        === ":fontawesome-brands-apple: macOS"
            TBA

        === ":fontawesome-brands-windows: Windows"
            TBA

> - `LT_GPIO_DEV_PATH` (default: `"/dev/gpiochip0"`) to set the path to the GPIO device where TROPIC01's interrupt (INT) and Chip Select (CS) line is connected:

Reviewer: ```suggestion
- `LT_GPIO_DEV_PATH` (default: `"/dev/gpiochip0"`) specifies the path to the GPIO device to which TROPIC01's interrupt (INT) and Chip Select (CS) lines are connected:
```
